app/templates/macros/toolkit_forms.html

```diff
@@ -2,6 +2,7 @@
   {%
     with
     question=question_content.question,
+    hint=question_content.hint or None,
```

Comment:
Let's fix the toolkit first then. I'll look into it.

On 30 July 2015 at 14:24, Alexey Bezhan <notifications@github.com> wrote:

> In app/templates/macros/toolkit_forms.html
> <https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/130#discussion_r35869328>
> :
>
> > @@ -2,6 +2,7 @@
> >    {%
> >      with
> >      question=question_content.question,
> > +    hint=question_content.hint or None,
>
> or '', but toolkit really should check for {% if hint %} instead of if
> hint is defined, otherwise we'll get an empty hint span for all questions
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/130/files#r35869328>
> .
>